org.eclipse.scout.rt.rest/src/main/java/org/eclipse/scout/rt/rest/client/RestClientProperties.java

```diff
@@ -265,6 +266,18 @@ public enum LoggerVerbosity {
    */
   public static final String VALIDATE_CONNECTION_AFTER_INACTIVITY = "scout.rest.client.http.validateAfterInactivity";
 
+  /**
+   * Specifies the total span of time in milliseconds connections can be kept alive or execute requests. May be zero if the
+   * connection does not have an expiry deadline.
+   * <p>
+   * The value MUST be an instance convertible to {@link java.lang.Long}.
+   * </p>
+   * <p>
+   * The default value is 30 * 60 * 1000 (e.g. 30 minutes).
```

Comment:
Is this correct? Isn't the default value defined by the config property (which could be set to another value)?

Reply:
Adjusted comment.